servers/mlflowserver/mlflowserver/conda_env_create.py

```diff
@@ -0,0 +1,155 @@
+"""
+This script creates a Conda environment from a `conda.yaml`.
+The `conda.yaml` file gets fetched from cloud storage.
+"""
+import os
+import logging
+import argparse
+import json
+import yaml
+import tempfile
+from subprocess import run
+from seldon_core.microservice import PARAMETERS_ENV_NAME, parse_parameters
+from seldon_core import Storage
+from shlex import quote
```

Comment:
Is shlex part of the python package, or do we need to add this as a dependency?

Reply:
It seems to be part of the [standard library](https://docs.python.org/3/library/shlex.html). Nothing you can't find on Python's standard library!